tests/common/test_case.py

```diff
@@ -269,23 +269,38 @@ def write_tuple_inference_files(self):
                              'ss ss<>ve gg<>o sd<>ccs\t1\n')
 
     def write_span_prediction_files(self):
-        with codecs.open(self.VALIDATION_FILE, 'w', 'utf-8') as train_file:
-            train_file.write('1\tquestion 1 with extra words\tpassage with answer and a reallylongword\t13,18\n')
+        with codecs.open(self.VALIDATION_FILE, 'w', 'utf-8') as validation_file:
+            validation_file.write('1\tquestion 1 with extra words\t'
+                                  'passage with answer and a reallylongword\t13,18\n')
         with codecs.open(self.TRAIN_FILE, 'w', 'utf-8') as train_file:
             train_file.write('1\tquestion 1\tpassage1 with answer1\t14,20\n')
             train_file.write('2\tquestion 2\tpassage2 with answer2\t0,8\n')
             train_file.write('3\tquestion 3\tpassage3 with answer3\t9,13\n')
             train_file.write('4\tquestion 4\tpassage4 with answer4\t14,20\n')
 
     def write_additional_span_prediction_files(self):
-        with codecs.open(self.VALIDATION_FILE, 'w', 'utf-8') as train_file:
-            train_file.write('1\tquestion 2\tpassage with perhaps the answer\t13,18\n')
+        with codecs.open(self.VALIDATION_FILE, 'w', 'utf-8') as validation_file:
+            validation_file.write('1\tquestion 2\tpassage with perhaps the answer\t13,18\n')
         with codecs.open(self.TRAIN_FILE, 'w', 'utf-8') as train_file:
             train_file.write('1\tquestion 5\tpassage5 with answer5 answer3\t14,20\n')
             train_file.write('2\tquestion 6\tpassage6 with answer6\t0,8\n')
             train_file.write('3\tquestion 7\tpassage7 with answer7\t9,13\n')
             train_file.write('4\tquestion 8\tpassage8 with answer8\t14,20\n')
 
+    def write_sentence_selection_files(self):
+        with codecs.open(self.VALIDATION_FILE, 'w', 'utf-8') as validation_file:
+            validation_file.write('1\tWhere is Paris?\tParis is the capital of France.###It '
```

Comment:
Can you make this have four sentences, with one of them really long?  Just to be sure padding is set correctly on model load.